### PR TITLE
fix emit code

### PR DESCRIPTION
**Pull request by jsalant22**
Undo changes that inadvertantly reverted emit code

### CODE REVIEW

File: _unittest/test_26_emit.py [hunk 1]
```diff
@@ -650,11 +643,11 @@ def test_N_to_1_feature(self):
         assert instance.get_largest_problem_type(econsts.result_type().emi) == "Out-of-Channel: Tx Fundamental"
         domain2 = self.aedtapp.results.interaction_domain()
         rx_frequencies = rev.get_active_frequencies(radiosRX[0], bandsRX[0], econsts.tx_rx_mode().rx, "Hz")
-        domain2.set_receiver(radiosRX[0], bandsRX[0], rx_frequencies[0])
+        domain2.set_receiver(radiosRX[0], bandsRX[0], rx_frequencies[0], "MHz")
```
Comment (apollo-crum) [inline _unittest/test_26_emit.py:646]: Are we retrieving rx_frequencies in Hz but setting them in MHz?  Original change just used Hz.

**Review by maxcapodi78 — APPROVED** [review]
LGTM

File: pyaedt/emit_core/results/revision.py [hunk 2]
```diff
@@ -320,7 +287,7 @@ def get_active_frequencies(self, radio_name, band_name, tx_rx_mode, units=""):
 
         Returns
         -------
-        freq:class:`list of float`
+        freqs:class:`list of float`
```
Comment (MaxJPRey) [inline pyaedt/emit_core/results/revision.py:290]:
```suggestion
        freqs : List of float
```

File: _unittest/test_26_emit.py (hunk 1 above)
Comment (jsalant22) [reply _unittest/test_26_emit.py]: @apollo-crum good catch

**Review by apollo-crum — APPROVED** [review]

**Pull request merged**